.devcontainer/on_amd64/devcontainer.json

```diff
@@ -0,0 +1,64 @@
+// https://github.com/athackst/vscode_ros2_workspace
```

Comment:
new dev container for desktop, that shouldn't use host networking and elevated permissions (and GPU)